packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,133 @@
+    is_dialog_open: boolean;
+    is_drawer_open: boolean;
+    is_running: boolean;
+    is_stop_button_disabled: boolean;
+    is_stop_button_visible: boolean;
+    closeResetDialog: () => void;
+    onOkButtonClick: () => void;
+    onResetClick: () => void;
+    onRunButtonClick: () => void;
+    onSortClick: () => void;
+    onUndoClick: () => void;
+    onZoomInOutClick: () => void;
+    toggleSaveLoadModal: () => void;
+    is_mobile: boolean;
+    toggleStrategyModal: () => void;
+    toggleLoadModal: () => void;
+    toggleSaveModal: () => void;
+};
+
+const Toolbar = (props: TToolbar) => {
+    const {
+        is_mobile,
+        is_running,
+        active_tab,
+        is_dialog_open,
+        onOkButtonClick,
+        closeResetDialog,
+        toggleStrategyModal,
+        toggleLoadModal,
+        toggleSaveModal,
+    } = props;
+
+    return (
+        <>
+            {is_mobile ? (
+                <div className='toolbar'>
+                    <div className='toolbar__section'>
+                        <ToolbarButton
+                            button_id='db-toolbar__import-button--mobile'
+                            button_classname='toolbar__btn--icon'
+                            buttonOnClick={toggleLoadModal}
+                            icon={<Icon icon='IcFolderOpenFilled' color='active' />}
+                            button_text={localize('Load')}
+                        />
+                        <ToolbarButton
+                            button_id='db-toolbar__quick-strategy-button--mobile'
+                            button_classname='toolbar__btn--icon'
+                            buttonOnClick={toggleStrategyModal}
+                            icon={<Icon icon='IcPuzzle' color='active' />}
+                            button_text={localize('Quick')}
+                        />
+                        <ToolbarButton
+                            button_id='db-toolbar__save-button--mobile'
+                            button_classname='toolbar__btn--icon'
+                            buttonOnClick={toggleSaveModal}
+                            icon={<Icon icon='IcSaveFilled' color='active' />}
+                            button_text={localize('Save')}
+                        />
+                    </div>
+                </div>
+            ) : (
+                <div className='toolbar dashboard__toolbar'>
+                    <div className='toolbar__section'>
+                        {active_tab === tabs_title.WORKSPACE && <WorkspaceGroup {...props} />}
+                    </div>
+                </div>
+            )}
+            <SaveModal />
+            <LoadModal />
+            <Dialog
+                title={localize('Are you sure?')}
+                is_visible={is_dialog_open}
+                confirm_button_text={is_running ? localize('Yes') : localize('OK')}
+                onConfirm={onOkButtonClick}
+                cancel_button_text={is_running ? localize('No') : localize('Cancel')}
```

Comment:
Let's seperate the logic from here and use as a variable

Reply:
done ✅